app/src/main/java/com/kickstarter/libs/AnalyticEvents.kt

```diff
@@ -964,6 +965,13 @@ class AnalyticEvents(trackingClients: List<TrackingClientType?>) {
         client.track(CREATOR_DETAILS_CLICKED, experimentProperties(projectData))
     }
 
+    fun trackCreatorDetails(projectData: ProjectData) {
+        val props: HashMap<String, Any> = hashMapOf(CONTEXT_CTA.contextName to CREATOR_DETAILS.contextName)
+        props[CONTEXT_PAGE.contextName] = PROJECT.contextName
+        props.putAll(experimentProperties(projectData))
```

Comment:
one small thing here, the `experimentProperties()` function here is calling sending an event to optimizely, which we don't need for this. I would change this to `props.putAll(AnalyticEventsUtils.projectProperties(project, client.loggedInUser()))`

Reply:
Resolved